fea-rs/test-data/parse-tests/good/spec6iii_3c.fea

```diff
@@ -0,0 +1,5 @@
+# http://adobe-type-tools.github.io/afdko/OpenTypeFeatureFileSpecification.html#example-3c
+feature kern {
+    pos a b' c 10;
+    pos e f' g h <12 0 5 0>;
```

Comment:
I suggest adding comments to explain what these mean. They are fairly hard to read even with the adobe reference open. E.g. bc preceeded by a increasing the advance of c by 10 if my skim reading is correct

Reply:
It definitely takes a while to wrap your head around FEA, but I'd prefer to keep any serious exposition out of the test files, and particularly the parse tests, since comments that we include here also end up in the output.

In this case, each line is defining a chain rule. The funny thing about this particular syntax is that, although you provide a value record at the end of the line, the value record is actually applied to the marked glyph (marked with `'`) and so what this produces is a chain rule where if the glyph pointer is at `b`, and it is preceded by an `a` and followed by a `c`, it will apply a lookup that adds 10 to the x-advance of `b`.